cmake/SearchForStuff.cmake

```
@@ -164,9 +164,12 @@ if (PKG_CONFIG_FOUND)
   # Find Simbody
   set(SimTK_INSTALL_DIR ${SimTK_INSTALL_PREFIX})
   #list(APPEND CMAKE_MODULE_PATH ${SimTK_INSTALL_PREFIX}/share/cmake)
-  find_package(Simbody)
-  if (Simbody_FOUND)
-    message (STATUS "Looking for Simbody - found")
+  find_package(Simbody CONFIG)
```

Unless someone is shipping a `FindSimbody.cmake` and putting it in the `CMAKE_MODULE_PATH`, I guess there is no reason to explicitly ask for `CONFIG`,right?
```suggestion
  find_package(Simbody)
```

---

This is how it done for vcpkg.
I think it not find it without config.

---

I will try.

---

The vcpkg give the correct parameters to find it with config.
I will try without.

---

Unless a `FindSimbody.cmake` script is present somehow in `CMAKE_MODULE_PATH`, `find_package(Simbody)` should just fallback to `find_package(Simbody CONFIG)`, see https://cmake.org/cmake/help/latest/command/find_package.html for more info. vcpkg suggests `find_package(<pkg> CONFIG)` in its install message, but `find_package(<pkg>)` should work fine as well. For this reason, I would prefer to minimize the unneccessary changes and for consistency with the other dependencies, thanks!

---

ok. I already trying all your suggestion. Something is not working, and I am not sure what.

---

done.